music/src/core/player.ts

```diff
@@ -541,3 +541,134 @@ export class PlayerWithClick extends Player {
     super(true, callbackObject);
   }
 }
+
+/**
+ * A `NoteSequence` player that uses a MIDI output for playing. Note that
+ * WebMIDI is not supported in all browsers. A
+ * [polyfill](https://cwilso.github.io/WebMIDIAPIShim/) exists, but it too
+ * requires a plugin to be installed on the user's computer, so it might not
+ * work in all cases.
+ *
+ * If you want to use a particular MIDI output, you must update the `output`
+ * property before calling `start`, otherwise a message will be sent to
+ * all connected MIDI outputs:
+ *
+ * Example (easy mode):
+ *
+ *   `
+ *    const player = new mm.MIDIPlayer();
+ *    player.requestMIDIAccess().then(() => {
+ *      player.output = [player.midiOutputs[0]];
```

Comment:
does this only play on one of the outputs?

Reply:
in this example, yes. it's an array though, so you can pass a number of outputs (like in the example below)

Reply:
can you add to the end of the last line `# arbitrarily choose first port`

Reply:
actually, isn't it easier (for easy mode) just to send it to all ports?

Reply:
sure!